examples/getting-started-movielens/01-Download-Convert.ipynb

```diff
@@ -58,40 +58,20 @@
     "from sklearn.model_selection import train_test_split"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "'0.18.0'"
-      ]
-     },
-     "execution_count": 3,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "cudf.__version__"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We define our base directory, containing the data."
+    "We define our base input directory, containing the data."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "BASE_DIR = '~/nvt-examples/movielens/data/'"
+    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '/model/data/')"
```

Comment:
I don't love the default path being ```/model/data``` - I think we should minimally include the datasetname in the default path being used.

How about we use the previous default? :
```suggestion
    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '~/nvt-examples/movielens/data/')"
```